docs/docs/schema/config.json

```diff
@@ -200,6 +212,36 @@
             "title": "PipelineConfig",
             "type": "object"
         },
+        "SchemaRegistryConfig": {
+            "additionalProperties": false,
+            "description": "Base class for settings, allowing values to be overridden by environment variables.\n\nThis is useful in production for secrets you do not wish to save in code, it plays nicely with docker(-compose),\nHeroku and any 12 factor app design.",
+            "properties": {
+                "enabled": {
+                    "default": false,
+                    "description": "If the Schema Registry handler should be initialized.",
+                    "env_names": [
+                        "enabled"
+                    ],
+                    "title": "Enabled",
+                    "type": "boolean"
+                },
+                "url": {
+                    "default": "http://localhost:8081",
```

Comment:
why is it named host for Kafka REST Proxy & Kafka Connect but URL for Schema Registry? AFAICT these have the same format

Reply:
Yeah, I know I was not sure which one to choose tbh. We should either stick with `url` or `host`. What do you prefer?

Reply:
if we use the `HttpUrl`/`AnyUrl` type from Pydantic, I would prefer URL